City-of-Helsinki/drupal-hdbt-admin
# UHF-X Drupal core 9.4.0 related bug fixes

## What was done
* Fixed paragraph adding button width which was set to fixed width in Drupal core 9.4.0 Claro update.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT Admin theme
    * `composer require drupal/hdbt_admin:dev-UHF-X_drupal_core_940_related`
* Run `make drush-cr`

## How to test
* Check that the width of paragraph selection is 100% and the layout bug (see below) is fixed

   * ![image](https://user-images.githubusercontent.com/1712902/174264139-cbffb168-55c2-451b-b0bd-6efec061704c.png)
